cc.config.json

```diff
@@ -8,9 +8,7 @@
                 "970b0598-bcb0-4714-91fb-2e81440dccd8",
                 "bcd64cc6-2dd9-43f6-abbe-66318d332032",
                 "d930590d-bb92-4cc8-8bd1-23cd027f9edf",
-                "a3cd009f-0ab0-420d-9278-b9fdab939bbc",
-                "5c601d96-e4c7-4698-991b-7ee674b11079",
```

Comment:
"5c601d96-e4c7-4698-991b-7ee674b11079" is "copy-pass.effect",move to CustomPipeline